core-bundle/src/Resources/contao/pages/PageError401.php

```diff
@@ -31,8 +31,6 @@ class PageError401 extends Frontend
 	 */
 	public function generate($objRootPage=null)
 	{
-		@trigger_error('Using PageError401::generate() has been deprecated in Contao 4.9 and will be removed in Contao 5.0. Use the PageError401::getResponse() method instead.');
```

Comment:
Looks like I forgot `E_USER_DEPRECATED` constant is missing, maybe that's the original issue? 😆

Reply:
See 4cec36c79219b1c2efe10b69c6f4f2f30744bd5a

Reply:
how did you find that commit, since it doesn't belong to this branch? Was it a previous commit/attempt Github isn't showing?

Reply:
<img width="1125" alt="Bildschirmfoto 2022-03-28 um 09 16 44" src="https://user-images.githubusercontent.com/367169/160345942-bf131649-1c15-4c46-968b-422783b3143e.png">

It is the first commit in this PR?

Reply:
See https://github.com/contao/contao/pull/4252#issuecomment-1079157780

> I had to remove the `trigger_error()` calls because there is no `$this->expectDeprecation()` method in the Symfony PHPUnit bridge in version 4.4 and thus I cannot fix the unit tests otherwise.